docs/api/feedback.md

```diff
@@ -42,13 +42,49 @@ Content-Type: audio/ogg
 
 <audio blob>
 --YvHKkjjzXfysYJVHMoOAoNczae
-Content-Disposition: form-data; name="errors"
+Content-Disposition: form-data; name="sentences"
+Content-Type: application/json
 
-0
+{
+  "sentence": 0,
+  "errors": 1,
+  "words": [
+    {
+      "expected": "hello",
+      "recognized": "hello",
+      "error": null,
+      "correct": true
+    },
+    {
+      "expected": "there",
+      "recognized": "their",
+      "error": "substitution",
+      "correct": false
+    }
+  ]
+}
 --YvHKkjjzXfysYJVHMoOAoNczae
-Content-Disposition: form-data; name="errors"
+Content-Disposition: form-data; name="sentences"
+Content-Type: application/json
 
-2
+{
+  "sentence": 1,
+  "errors": 0,
+  "words": [
+    {
+      "expected": "general",
+      "recognized": "general",
+      "error": null,
+      "correct": true
+    },
+    {
+      "expected": "kenobi",
+      "recognized": "kenobi",
+      "error": null,
+      "correct": true
+    }
```

Comment:
We know that we want timing information, can we add it already in the docs?
I was thinking about `recognitionStart` and `recognitionEnd` but it could be simpler maybe...

Reply:
I don't think that I can add timing information yet as I don't have a clue about the internal format. I asked about this and am waiting for a reply. As soon as I know more I'll update the docs.

Reply:
Timing will not be added in the client results so this isn't relevant any more.